eclipse/sw360
# feat(ui): Added support to add external URLs to project

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.

* Removed "homepage", "wiki" and migrated it to "externalUrls" in project structure.
* Users now can add any number of urls(for e.g. "clearing" URL) in project through newly added "External Urls" section in UI.
* REST: Have kept the response containing "homepage" & "wiki" while getting projects with all details as it is.
  Have kept the newly added "exteralUrls" not readable and writable from REST for now as it was there for
  "homepage" URL and "wiki" URL.

> * Which issue is this pull request belonging to and how is it solving it? (*#1255*)
> * Did you add or update any new dependencies that are required for your change? - No

Issue: Closes #1255 

### Suggest Reviewer
> You can suggest reviewers here with an @mention.

### How To Test?
> How should these changes be tested by the reviewer?

* Run the migration scripts to verify migration of existing "homepage" and "wiki" to "externalUrls" and deletion of fields("wiki" and "homepage") after migration.
* Need to verify adding, editing, deleting external urls in the UI section of the project. 

> Have you implemented any additional tests? - No

### Checklist
Must:
